src/libraries/System.Net.Ping/tests/FunctionalTests/PingTest.cs

```diff
@@ -814,25 +819,21 @@ public void SendPing_LocaleEnvVarsMustBeIgnored(AddressFamily addressFamily, str
         public void SendPingAsync_LocaleEnvVarsMustBeIgnored(AddressFamily addressFamily, string envVar_LANG, string envVar_LC_MESSAGES, string envVar_LC_ALL)
         {
             IPAddress localIpAddress = TestSettings.GetLocalIPAddress(addressFamily);
-            if (localIpAddress == null)
```

Comment:
why did you remove this?

Reply:
To my knowledge because `localIpAddress` is never null.